playbooks/common/openshift-master/config.yml

```diff
@@ -313,6 +313,7 @@
 
 - name: Configure master instances
   hosts: oo_masters_to_config
+  any_errors_fatal: true
```

Comment:
Calling attention to this line. This will cause any failures in the `openshift_master` role to halt execution on all hosts. I think this makes sense to do for masters but a node failure doesn't seem fatal.